test/system/avo/rubygems_test.rb

```diff
@@ -0,0 +1,85 @@
+require "application_system_test_case"
+
+class Avo::RubygemsSystemTest < ApplicationSystemTestCase
+  def sign_in_as(user)
+    OmniAuth.config.mock_auth[:github] = OmniAuth::AuthHash.new(
+      provider: "github",
+      uid: "1",
+      credentials: {
+        token: user.oauth_token,
+        expires: false
+      },
+      info: {
+        name: user.login
+      }
+    )
+    @octokit_stubs.post("/graphql",
+      {
+        query: GitHubOAuthable::INFO_QUERY,
+        variables: { organization_name: "rubygems" }
+      }.to_json) do |_env|
+      [200, { "Content-Type" => "application/json" }, JSON.generate(
+        data: user.info_data
+      )]
+    end
+
+    visit avo.root_path
+    click_button "Log in with GitHub"
+
+    page.assert_text user.login
+  end
+
+  test "release reserved namespace" do
+    Minitest::Test.make_my_diffs_pretty!
```

Comment:
Is this needed? What about to move it into `test_helper` at least?

Reply:
It's not really required. Was used to see the diff. Removed here a7fdb6a61d2a9b5c3a846e966ae7e22216b229ef